docs/install.md

```diff
@@ -1,77 +1,197 @@
+    *   `Cloud Datastore User`: Used by PSQ to store result data, and in the
+        future by the Task Manager to store queriable task data
+    *   `Pub/Sub Editor`: Used by clients to talk to Turbinia, and by the Task
+        Manager to talk to workers
+    *   `Storage Object Admin` and `Storage Legacy Bucket Reader`: Only required
+        on the GCS bucket used by Turbinia, if any. See [GCP Turbinia](#gcp-turbinia)
+        for details
+    *   `Compute Instance Admin`: Used to list instances and to attach disks to
+        instances
+    *   `Service Account User`: Used when attaching disks
+    *   `Cloud Functions Developer`: Used by turbiniactl to query task status
+*   Create a new key for your service account, and then point to it with an
+    environment variable:
+    *   `export
+        GOOGLE_APPLICATION_CREDENTIALS="/home/turbinia/turbinia-service-account-creds.json"`
+*   Add the service account to the gcloud auth **(RECOMMENDED)**
+    *   `gcloud auth list`
+    *   `gcloud auth activate-service-account
+        --key-file=$GOOGLE_APPLICATION_CREDENTIALS`
+*   Alternately you can run Turbinia under your own credentials **(NOT
+    RECOMMENDED)**
+    *   Run `gcloud auth login` (may require you to copy/paste url to browser)
+    *   Or run `gcloud auth application-default login`
+*   Continue to [Build and Configure](#build-and-configure)
+
+#### Build and Configure
+
+*   Install Turbinia
+    *   `sudo apt-get install liblzma-dev`
+    *   `git clone https://github.com/google/turbinia.git`
+    *   `pip install --upgrade pip`
+    *   `pip install -r turbinia/requirements.txt`
+*   Install Plaso
+    *   `curl -o plaso-requirements.txt
+        https://raw.githubusercontent.com/log2timeline/plaso/master/requirements.txt`
+    *   `pip install -r plaso-requirements.txt`
+*   Update Python Path
+    *   Until [Issue #39](https://github.com/google/turbinia/issues/39) is
+        fixed, we need to adjust the `$PYTHONPATH` from the root of the Turbinia
+        repository:
+        *   ``cd turbinia; export PYTHONPATH=$PYTHONPATH:`pwd`; cd -``
+    *   And set it in your .bashrc file:
+        *   ``cd turbinia; echo "export PYTHONPATH=$PYTHONPATH:`pwd`" >>
+            ~/.bashrc ; cd -``
+*   Choose one of the locations for storing the Turbinia config and configure
+    from there:
+    *   `/etc/turbinia/` **(RECOMMENDED)**
+        *   `sudo mkdir /etc/turbinia`
+        *   `cp <localgitpath>/turbinia/config/turbinia_config.py /etc/turbinia/`
+    *   `/home/turbinia/.turbinia`
+        *   `cp <localgitpath>/turbinia/config/turbinia_config.py /home/turbinia/.turbiniarc`
+    *   Directly configure `<localgitpath>/turbinia/config/turbinia_config.py`
+    *   ***NOTE***: Match the `PUBSUB_TOPIC` variable in the configuration to the name
+        of the topic and subscription you created in the GCP.
+*   Continue to [Deploy the Cloud Functions](#deploy-the-cloud-functions)
+
+### Deploy the Cloud Functions
```

Comment:
Maybe just inline this section with the section that references it since it's so short?

Reply:
done